tests/e2e/fullsync_test_for_block_volume.go

```diff
@@ -97,6 +98,7 @@ var _ bool = ginkgo.Describe("full-sync-test", func() {
 		bootstrap()
 		scParameters = make(map[string]string)
 		storagePolicyName = GetAndExpectStringEnvVar(envStoragePolicyNameForSharedDatastores)
+		busyBoxImageOnGcr = GetAndExpectStringEnvVar(busyBoxImage)
```

Comment:
set this variable only once, may be in suite_test.go and in e2e_common.go
replace all usage of `busyBoxImageOnGcr` with `busyBoxImage`